### PR TITLE
document user_config_dir in README

### LINKED ISSUE #149: Documentation examples for `user_config_dir`
The only example of a `user_config_dir` value int he read-me file is for Linux. Would be nice to have examples for at least Windows and Mac as well so we can tell potential users where they should be looking for an application's configuration files.

Thanks a lot for this module! Just found about it today and it saved me a lot of time compared to cooking up a cross-platform solution on my own!

### PR DESCRIPTION
**Pull request by untzag**
closes #149 

https://github.com/ActiveState/appdirs/blob/master/appdirs.py#L186-L189

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -61,6 +61,8 @@ On macOS::
     '/Users/trentm/Library/Caches/SuperApp'
     >>> user_log_dir(appname, appauthor)
     '/Users/trentm/Library/Logs/SuperApp'
+    >>> user_config_dir(appname)
+    '/Library/Preferences/SuperApp'
```
Comment (harrybiddle) [inline README.rst:65]:
```suggestion
    '/Users/trentm/Library/Application Support/SuperApp'
```

File: README.rst [hunk 2]
```diff
@@ -61,6 +61,8 @@ On macOS::
     '/Users/trentm/Library/Caches/SuperApp'
     >>> user_log_dir(appname, appauthor)
     '/Users/trentm/Library/Logs/SuperApp'
+    >>> user_config_dir(appname)
```
Comment (harrybiddle) [inline README.rst:64]:
```suggestion
    >>> user_config_dir(appname, appauthor)
```

**Review by harrybiddle — CHANGES_REQUESTED** [review]

**Comment by harrybiddle** [comment]
Awesome. I can't test the Windows ones but they look good to me (the code mentions they will be the same as the data dir). I can however correct the Mac ones.

**Comment by untzag** [comment]
@harrybiddle I was confused by your observed behavior, so I looked into it a bit more and it turns out that OSX has changed since the last release :confounded: 

https://github.com/ActiveState/appdirs/commit/c99f3303007a1803cbaa23fc2830336ec551460b

I guess we're gonna need a maintainer to figure this out for us...

**Comment by harrybiddle** [comment]
Oh, dear :(

**Comment by krystian3w** [comment]
#160

**Pull request closed without merging**